Development/nmos/system_resources.cpp

```diff
@@ -1,21 +1,37 @@
 #include "nmos/system_resources.h"
 
+#include "cpprest/json_validator.h"
 #include "nmos/is09_versions.h"
+#include "nmos/json_schema.h"
 #include "nmos/json_fields.h"
 #include "nmos/resource.h"
 
 namespace nmos
 {
     nmos::resource make_system_global(const nmos::id& id, const nmos::settings& settings)
     {
-        return{ is09_versions::v1_0, types::global, make_system_global_data(id, settings), true };
+        const auto version = is09_versions::v1_0;
+        const auto data = make_system_global_data(id, settings);
+
+        // ensure system global data is correctly formated
+        const web::json::experimental::json_validator validator
+        {
+            nmos::experimental::load_json_schema,
+            boost::copy_range<std::vector<web::uri>>(is09_versions::all | boost::adaptors::transformed(experimental::make_systemapi_global_schema_uri))
+        };
```

Comment:
I'd have liked to see this pulled out like this:
https://github.com/garethsb/nmos-cpp/blob/0757c01454c95777ab18951bc65669c34276f7a1/Development/nmos/node_system_behaviour.cpp#L266-L275

Reply:
done